### ISSUE TITLE
Brickman and lms2012 are competing for display

### ISSUE THREAD

**Issue by laurensvalk** [ev3dev/lms2012-compat#60]
The brickman display sometimes returns after a few seconds of inactivity. When I press a button, the lms2012 menu returns.

I'm running the latest stretch image of July 25 2017, with no modification except the steps listed in the readme.

```
Linux ev3dev 4.9.39-ev3dev-1.3.0-ev3 #1 PREEMPT Tue Jul 25 15:08:42 CDT 2017 armv5tejl GNU/Linux
```

**Comment by WasabiFan** [comment]
:laughing:

https://github.com/ev3dev/lms2012-compat/issues/59#issuecomment-317904196

**Comment by dlech** [comment]
Yes, this is a bug in Brickman on Stretch.

**Comment by laurensvalk** [comment]
Oops, sorry.

**Comment by dlech** [comment]
brickman is finally fixed in `brickman 0.10.0`
